test/sagemaker_tests/mxnet/training/integration/local/test_mnist_training.py

```diff
@@ -37,6 +38,8 @@ def test_single_machine(docker_image, sagemaker_local_session, local_instance_ty
     _train_and_assert_success(mx, str(tmpdir))
 
 
+@pytest.mark.model("mnist")
+@pytest.mark.multinode("multinode")
```

Comment:
There is a pytest.skip inside the function for some reason.

Could you add a pytest.mark.processor("cpu") here?